index.js

```diff
@@ -23,70 +26,84 @@ const development = false;
 
 const parsePost = (req, res) => {
 	const post = new posts.Post(req.body.body, req.body.title, req.body.tags, req.body.description);
-	console.log(`Created post #${post.id}`);
 	post.save();
-	res.send('OK')
+	res.send('OK');
+	console.log(`Created post #${req.body.title}`);
 }
 
 const getPost = (req, res) => {
 	const postId = req.query.id;
-	const post = posts.getPostMetadata()[postId - 1];
-	post.body = posts.getBody(post.id)
-	// see what post number is being requested
-	console.log("Post ID Number: " + postId)
-	res.send(post);
+	var client = new MongoClient(mongodb_foo.url)
```

Comment:
Why create a new client when getting post? Isnt this inneficient?

Reply:
the idea was to have a client running only when it is needed